test/test_helper.rb

```diff
@@ -1,6 +1,8 @@
 ENV['RAILS_ENV'] ||= 'test'
 require File.expand_path('../../config/environment', __FILE__)
 require 'rails/test_help'
+require 'capybara/rails'
+require 'mocha/mini_test'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.